fabric-network/test/transaction.js

```diff
@@ -281,6 +281,12 @@ describe('Transaction', () => {
 			return expect(promise).to.be.eventually.rejectedWith(status).and.have.deep.property('responses', [validEnsorsementResponse]);
 		});
 
+		it.only('throws with peer responses if the orderer returns an undefined response', async () => {
```

Comment:
I'm not sure if a test is needed at this level. Certainly need to remove the `.only`. What is here now is faking out so much it's difficult to see any value.